components/edit-collective/sections/Updates.js

```diff
@@ -81,7 +81,9 @@ const Updates = ({ collective }) => {
             isLoading={loading}
             onChange={({ target }) => {
               const updatedCollective = cloneDeep(collective);
-              set(updatedCollective, FEATURE_FLAGS.UPDATES, target.value);
+              const sections = get(updatedCollective, 'settings.collectivePage.sections');
```

Comment:
Same here, if the toggle on "Edit collective page" changes the feature status, we should just remove this standalone page.